src/main/java/org/broadinstitute/consent/http/models/DACUser.java

```diff
@@ -1,12 +1,22 @@
 package org.broadinstitute.consent.http.models;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.ArrayList;
 import java.util.Date;
 import java.util.List;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonParser;
 import org.apache.commons.lang3.builder.EqualsBuilder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class DACUser {
 
+    private static final Logger logger = LoggerFactory.getLogger(DACUser.class.getName());
```

Comment:
Shouldn't need the call to `.getName()` here -- I prefer to pass the class in and let slf4j figure out the formatting.

Reply:
Good catch - will double check that that works.